templates/projects/empty-server/data.js

```diff
@@ -79,7 +79,8 @@ template.server = {
       },
       cors: false,
       errorHandler: {
-        disableStackTrace: false,
+        debug: true,
```

Comment:
On the second thought, I think it may be best to disable the built-in error handler in strong-remoting and always use strong-error-handler instead.

```diff
   cors: false
-  errorHandler: {
-    disableStackTrace: false
- },
+  handleErrors: false,
 }},
```

That way we don't have to duplicate the configuration of strong-error-handler in multiple places.